src/Form.js

```diff
@@ -7,7 +8,7 @@ export default class Form {
      * @param {Function} rejected - function to call after reject
      * @memberof Form
      */
-    constructor({ inputs, form }, rejected = () => {}) {
+    constructor({ inputs, form }, rejected = () => {}, config = {alwaysCancelSubmit : true}) {
```

Comment:
Reject and config put in option: `constructor({ inputs, form, rejected, config })`